app/controllers/users_controller.rb

```diff
@@ -1,16 +1,30 @@
 class UsersController < ApplicationController
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
+  before_action :logged_in_user, except: %i(show new create)
+  before_action :load_user, except: %i(new create index)
```

Comment:
```suggestion
  before_action :load_user, except: %i(new create index)
```
index -> cái này cần đâu bé

Reply:
dạ e để except mà chị